### PR TITLE
feat: add air quality index

### LINKED ISSUE #43: Display Air Quality Index
Will show the Air Quality of a location by using the Open Weather API.

### PR DESCRIPTION
**Pull request by Keshraf**
- Added a indicator showing air quality

Closes #43

### CODE REVIEW

**Review by mukhsadr — APPROVED** [review]
LGTM!

File: src/components/AQICard/styles.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+.aqi-indicator-circle {
+  width: 6px;
+  height: 6px;
+  border-radius: 50%;
+  background-color: #fff;
+}
+
+.aqi-indicator-text {
```
Comment (mukhsadr) [inline src/components/AQICard/styles.css:70]: can you also give background for text so when the background of the app changes, the text is still visible!

image

**Review by mukhsadr — APPROVED** [review]
It looks good to me. After my review, I can say that air quality feature works fine. Indicator is changing accordingly to data changes and text is bold and visible in various background. @sumana2001 I think it is good to merge it now.

**Review by sumana2001 — CHANGES_REQUESTED** [review]
One small addition, could you add a title or a text saying air quality or something so that users know what scale it is showing

**Review by sumana2001 — COMMENTED** [review]
Since there is no change in package.json, please remove any changes made to yarn.lock and fix merge conflicts and then I think we are good to go

**Comment by Keshraf** [comment]
> Since there is no change in package.json, please remove any changes made to yarn.lock and fix merge conflicts and then I think we are good to go

I have resolved the merge conflicts and added the heading. The yarn.lock was added to the PR when I initially fixed the merge conflicts.

**Review by sumana2001 — CHANGES_REQUESTED** [review]
Seems like all the checks are failing. Please check the deploy log and fix the issue

**Comment by Keshraf** [comment]
> Seems like all the checks are failing. Please check the deploy log and fix the issue

The problem was in the yarn.lock file. Had to ```rm -rf node_modules``` and then reinstall all the packages ```yarn install```

**Comment by Keshraf** [comment]
@sumana2001 please review and merge this PR. I have solved merge conflicts thrice since the morning 🥲

**Comment by sumana2001** [comment]
Oh I am so sorry. This is the last time. let me know when you're done, I'll merge your PR first

**Comment by Keshraf** [comment]
@sumana2001, you can review it and merge the PR. I am done with it.

**Comment by sumana2001** [comment]
I can still see App.js in conflicting

**Comment by Keshraf** [comment]
@sumana2001 fixed now. I think I have become experienced in solving merge conflicts.

**Comment by sumana2001** [comment]
Haha great hob! That's the goal💯

**Pull request merged**